src/sentry/sentry_metrics/utils.py

```diff
@@ -14,17 +15,17 @@ class MetricIndexNotFound(InvalidParams):  # type: ignore
     pass
 
 
-def reverse_resolve(index: int) -> str:
+def reverse_resolve(use_case_id: UseCaseKey, index: int) -> str:
```

Comment:
the idea is very simple, right after we merge this PR we can create follow up PR to remove default value from `resolve` and `reverse_resolve` and i'm going to make it in similar manner as `bulk_record`:

https://github.com/getsentry/sentry/blob/d23fb690138b93944c9a410a3ba6b5e2ab7005e2/src/sentry/sentry_metrics/indexer/postgres_v2.py#L47

and `record`:

https://github.com/getsentry/sentry/blob/d23fb690138b93944c9a410a3ba6b5e2ab7005e2/src/sentry/sentry_metrics/indexer/postgres_v2.py#L166-L166

Also when it's first required parameter you can easily use `functools.partitial`.

Reply:
Ah I think I understand, you are saying that it is currently inconsistent anyways, and once you remove the default in the follow up PR, you will move `use_case_id` to first parameter anyways in those functions so that they are all consistent with `use_case_id` as the first argument?

Reply:
Okay that's fine then. Can we maybe add a ToDo comment with that so we don't forget?